UPGRADE-3.x.md

```diff
@@ -1,6 +1,10 @@
 UPGRADE 3.x
 ===========
 
+## Deprecated ModelsToArrayTransformer::$choiceList property
+
+ModelsToArrayTransformer::__construct() can be used with two parameters ($modelManager, $class) or three ($choiceList, $modelManager, $class )
```

Comment:
I don't really know since the behavior is the same as it was there is nothing to do differently